File: src/main/java/com/github/simiacryptus/aicoder/openai/Response.java
package com.github.simiacryptus.aicoder.openai;

public class Response {
    @SuppressWarnings("unused")
    public String object;
    @SuppressWarnings("unused")
    private Engine[] data;

    @SuppressWarnings("unused")
    public Response() {
    }

    @SuppressWarnings("unused")
    public Response(String object, Engine[] data) {
        this.object = object;
        this.data = data;
    }

}

File: src/main/kotlin/com/github/simiacryptus/aicoder/openai/core/Response.kt
package com.github.simiacryptus.aicoder.openai.core

class Response {
    @Suppress("unused")
    var `object`: String? = null

    @Suppress("unused")
    private var data: Array<Engine> = arrayOf()

    @Suppress("unused")
    constructor()

    @Suppress("unused")
    constructor(`object`: String?, data: Array<Engine>) {
        this.`object` = `object`
        this.data = data
    }
}